index.html

```diff
@@ -15,15 +15,148 @@
 
     <!-- We are using a CSS Reset file to standardize browser default css -->
     <link rel="stylesheet" href="reset.css" />
-
-    <!-- This is where all of our awesome CSS code goes -->
     <link rel="stylesheet" href="style.css" />
+    <link href="Mina/Mina-Regular.ttf" rel="stylesheet"> <!-- this is the logo-->
   </head>
-  <body>
-    <!--
-
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+  <main>
```

Comment:
All of your content has to go in the `body` element. 

Please have a look at the HTML structural exercises that we did for proper document structure

```html
<html>
<head>...</head>
<body>
  <header>...</header>
  <main>...</main>
  <footer>...</footer>
</body>
<html>